pkgs/servers/nginx-sso/default.nix

```diff
@@ -1,19 +1,24 @@
-{ buildGoPackage, fetchFromGitHub, lib, nixosTests }:
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+}:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "nginx-sso";
   version = "0.25.0";
-  rev = "v${version}";
-
-  goPackagePath = "github.com/Luzifer/nginx-sso";
 
   src = fetchFromGitHub {
-    inherit rev;
     owner = "Luzifer";
     repo = "nginx-sso";
-    sha256 = "0z5h92rpr1rcfk11ggsb9w4ipg93fcb9byll7vl4c0mfcqkpm2dr";
+    rev = "v${version}";
+    sha256 = "sha256-uYl6J2auAkboPpT6lRZzI70bCU9LvxfCdCyHfLNIsHw=";
   };
 
+  vendorSha256 = null;
+
+  doCheck = false;
```

Comment:
Is this just because the tests weren't ran before, or are they actually broken right now? If they are broken, could you document so in a comment for future readers? Thanks!

Reply:
I took another look that this failure and patched that case. All seems to be working now, including nixosTests.

Reply:
Seems like `buildGoPackages` didn't invoke tests at all before because the [change to `go vet`](https://go.dev/doc/go1.15#vet) was introduced a long time ago.